test-network-nano-bash/README.md

```diff
@@ -59,6 +59,16 @@ If you have trouble running bash scripts in your environment, you can just as ea
 
 Note the syntax of running the scripts. The peer admin scripts set the admin environment variables and must be run with the `source` command in order that the exported environment variables can be utilized by any subsequent user commands.
 
+## Running each component separately with CAs
+
+These instructions are for running the CAs from terminal sessions.  Open terminal windows for 3 CAs as seen in the following terminal setup. These instructions should be followd before opening the ordering and peer windows described above.
+
+![CA Terminal setup](ca_terminal_setup.png)
+
+- cd to the `test-network-nano-bash` directory in each terminal window
+- Instead of running the  `./generate_artifacts.sh` described above, in the three CA terminals, run `./ordererca.sh`, `./org1ca.sh`, `./org2ca.sh` respectively.
+- In the first orderer terminal, run `./generate_artifacts.sh "" true` to generate crypto material using the CAs and application channel genesis block and configuration transactions (calls configtxgen). The artifacts will be created in the `crypto-config` and `channel-artifacts` directories. If you are running BFT consensus then run `./generate_artifacts.sh BFT true`.  All artifacts generated with the CA will conform to the same directory structure as cryptogen.
```

Comment:
How about changing generate_artifacts to accept flags such as `-bft` and `-ca`? `generate_artifacts.sh "" true` isn't very expressive.

Reply:
this has been corrected.